vm/src/frame.rs

```diff
@@ -230,8 +230,8 @@ impl Frame {
                 Ok(None)
             }
             bytecode::Instruction::BuildSet { size, unpack } => {
-                let elements = self.get_elements(vm, *size, *unpack)?;
-                let py_obj = vm.ctx.new_set(elements);
+                let _elements = self.get_elements(vm, *size, *unpack)?;
+                let py_obj = vm.ctx.new_set();
```

Comment:
We need to pass in the elements here when using set literals like `{1,2,3}`. Please revert this change.

Reply:
Hmm, in retrospect it might be better to loop over the elements and call the set_add function with the vm, since you are right that the `__hash__` function requires the vm.

Reply:
Sounds good to me - might me missing why we need to do this, when it looks like I can initialize a set from an iterable without the change you suggest

Reply:
Okay, we can initialize from iterable, this would be best, but we require the usage of the `elements` in some way in order to construct set literals.